### DISCUSSION TITLE
when will we can use pdm create environment on Pycharm>interpreter

### DISCUSSION THREAD

**Discussion by shifenhutu** [pdm-project/pdm#2447] [Ideas]
路径 》 Add python interpreter

<img width="776" alt="image" src="https://github.com/pdm-project/pdm/assets/104620424/e2a58fe1-da1e-4b92-87cc-018f426073d4">

**Comment by pawamoy** [comment]
That's something to ask on PyCharm's bugtracker. If you do, please share the link here :slightly_smiling_face:

**Reply by frostming** [reply]
There is one: https://youtrack.jetbrains.com/issue/PY-49588/PDM-Support

**Reply by baggiponte** [reply]
I have been following that for waay to long. can't believe they haven't added support yet.
